packages/test-utils/src/instruments.mock.ts

```diff
@@ -5,6 +5,11 @@ import {
     PaymentInstrument,
 } from '@bigcommerce/checkout-sdk';
 
+export enum UntrustedShippingCardVerificationType {
```

Comment:
Unfortunately, I have to create this since the enum value is under the core directory which is not allowed to import from. 😢